source/icu.net/NativeMethods/NativeMethods.cs

```diff
@@ -90,16 +90,6 @@ static NativeMethods()
 		[DllImport(LIBDL_NAME, SetLastError = true)]
 		private static extern IntPtr dlsym(IntPtr handle, string name);
 
-		[DllImport(LIBDL_NAME, EntryPoint = "dlerror")]
-		private static extern IntPtr _dlerror();
-
-		private static string dlerror()
```

Comment:
This was used when logging load errors on Linux. It's not important in the successful cases, but I don't know how helpful it might be when debugging for some people/situations. Why did you remove it?

Reply:
Since it was no longer relevant in the `#if NET6_0_OR_GREATER` situation, and for the sake of having a common error line for all the OSes:
```
				lastError = Marshal.GetLastWin32Error();
				Trace.WriteLineIf(lastError != 0, $"Unable to load [{libPath}]. Error: {lastError}");
				Trace.TraceWarning($"{loadMethod} of {libPath} failed with error {lastError}");
```

Reply:
OS-specific error messages restored.